DependencyInjection/SonataMediaExtension.php

```diff
@@ -387,33 +388,58 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
 
         // add the default configuration for the S3 filesystem
         if ($container->hasDefinition('sonata.media.adapter.filesystem.s3') && isset($config['filesystem']['s3'])) {
+            $acl = str_replace(
```

Comment:
Why don't we use directly the correct value on our config instead of having to replace it here?

I mean, maybe for this PR we can do it in a BC way. Put a message here to remove this replaces, modify our documentation to show our new params, and maybe even trigger a deprecation message if someone use them?

WDYT @stloyd @sonata-project/contributors ?